model/portableElement/validator/Validator.php

```diff
@@ -184,6 +186,23 @@ public static function isValidVersion($value)
         return true;
     }
 
+    /**
+     * @param $value
+     * @return bool
+     * @throws PortableElementInvalidFieldException
+     */
+    public static function isValidSemVer($value)
+    {
+        $validator = \tao_helpers_form_FormFactory::getValidator(
+            self::Regex,
+            array('format' => '/^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*)?(\+[0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*)?$/')
```

Comment:
We are using this lib already: 
https://github.com/oat-sa/extension-tao-itemqti/blob/a4e4026602c78f22e99ce9a6a2c722ac180be794/composer.json#L61

Would it be possible to leverage it by reusing the helper https://github.com/naneau/semver/blob/master/src/Naneau/SemVer/Regex.php#L45-L66 ?

Reply:
done

Reply:
Ok, looks good :)